implementation/libs/jbb-lib-mvc/src/main/java/org/jbb/lib/mvc/repository/JbbSessionRepository.java

```diff
@@ -0,0 +1,75 @@
+package org.jbb.lib.mvc.repository;
+
+
+import org.springframework.session.ExpiringSession;
+import org.springframework.session.MapSession;
+import org.springframework.session.MapSessionRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+/**
+ *
+ * This is own implementation of MapSessionRepository. The only change with compare to original implementation is method @getSessionMap
+ * Method is using in administrator panel in session bookmark
+ *
+ */
+@Repository
+public class JbbSessionRepository extends MapSessionRepository{
```

Comment:
Extending `MapSessionRepository` is redundant if you are copy it to this class. Remove extending and just make this class implement `SessionRepository<ExpiringSession>` (just like `MapSessionRepository` does)